proto/redpanda/api/dataplane/v1alpha2/pipeline.proto

```diff
@@ -246,6 +246,24 @@ service PipelineService {
       }
     };
   }
+  rpc GetPipelineServiceConfigSchema(GetPipelineServiceConfigSchemaRequest) returns (GetPipelineServiceConfigSchemaResponse) {
+    option (google.api.http) = {
+      get: "/v1alpha2/redpanda-connect/config_schema"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Get Redpanda Connect pipeline service config schema"
```

Comment:
"pipeline service config schema" sounds weird to me. I'm unsure what the pipeline **service** refers to. How about this?

```suggestion
      summary: "Retrieve the schema for configuring Redpanda Connect pipelines."
```

Reply:
Updated.